src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionEasyOut.cs

```diff
@@ -26,8 +26,8 @@ static ConversionEasyOut()
                 const byte XRF = (byte)ConversionKind.ExplicitReference;
                 const byte XNM = (byte)ConversionKind.ExplicitNumeric;
                 const byte NOC = (byte)ConversionKind.NoConversion;
-                const byte BOX = (byte)ConversionKind.Boxing;
-                const byte UNB = (byte)ConversionKind.Unboxing;
+                const byte BOX = (byte)ConversionKind.ValueTypeBoxing;
+                const byte UNB = (byte)ConversionKind.ValueTypeUnboxing;
```

Comment:
note: this is the easy-out codepath.  It never considers type-parameters in its array.